src/core/reducers/api.js

```diff
@@ -7,8 +8,44 @@ import {
   SET_CLIENT_APP,
   SET_USER_AGENT,
 } from 'core/constants';
+import type {
+  SetAuthTokenAction,
+  LogOutUserAction,
+  SetClientAppAction,
+  SetLangAction,
+  SetUserAgentAction,
+} from 'core/actions/index';
+import type { Exact } from 'core/types/coreTypes';
 
-export default function api(state = {}, action) {
+type UserAgentInfoType = {|
+  browser: string,
+  os: string,
+|};
+
+export type ApiStateType = {
+  token: ?string,
+  lang: ?string,
+  clientApp: ?string,
+  userAgent: ?string,
+  userAgentInfo: ?UserAgentInfoType,
+};
+
+export const initialApiState = {
+  token: null,
+  lang: null,
+  clientApp: null,
+  userAgent: null,
+  userAgentInfo: null,
+};
+
+export default function api(
```

Comment:
The new docs are pretty nice. Were they not enough? Should I link to this [cheat sheet](http://www.saltycrane.com/blog/2016/06/flow-type-cheat-sheet/) ? I don't know how it's well it's maintained though.